### PR TITLE
chore: upgrade chromium to 76

### PR DESCRIPTION
**Pull request by MarshallOfSound**
As per @electron/wg-upgrades 

Notes: Upgraded Chromium to 76

### CODE REVIEW

File: atom/browser/ui/views/autofill_popup_view.cc [hunk 1]
```diff
@@ -150,7 +150,7 @@ void AutofillPopupView::OnSelectedRowChanged(
 
   if (current_row_selection) {
     int selected = current_row_selection.value_or(-1);
-    if (selected == -1 || selected >= child_count())
+    if (selected == -1 || static_cast<uint64_t>(selected) >= children().size())
```
Comment (nornagon) [inline atom/browser/ui/views/autofill_popup_view.cc]:
```suggestion
    if (selected == -1 || static_cast<size_t>(selected) >= children().size())
```

File: atom/common/platform_util_win.cc [hunk 2]
```diff
@@ -341,7 +341,7 @@ bool MoveItemToTrash(const base::FilePath& path) {
   // Elevation prompt enabled for UAC protected files.  This overrides the
   // SILENT, NO_UI and NOERRORUI flags.
 
-  if (base::win::GetVersion() >= base::win::VERSION_WIN8) {
+  if (base::win::GetVersion() >= base::win::Versionn::WIN8) {
```
Comment (deepak1556) [inline atom/common/platform_util_win.cc]:
```suggestion
  if (base::win::GetVersion() >= base::win::Version::WIN8) {
```

**Comment by deepak1556** [comment]
`76.0.3780.0` seems to be the latest canary, do we want to update here and merge ? Pretty sure there would be api and patch conflicts, I don't want electron-bot to handle it :)

**Comment by nornagon** [comment]
I think we should merge this first and then roll forward. I'd rather be on _any_ 76 version in 6-0-x ASAP.

**Comment by deepak1556** [comment]
cool, sounds fine.

**Pull request merged**